Documentation/Administration/Installation/ClassicMode/_codesnippets/_DockerDbDemo.sh

```diff
@@ -0,0 +1,6 @@
+docker run -d --name typo3db --network typo3-demo-net \
+    -e MYSQL_ROOT_PASSWORD=secret \
+    -e MYSQL_DATABASE=db \
+    -e MYSQL_USER=db \
+    -e MYSQL_PASSWORD=db \
+    mariadb
```

Comment:
You should specify a version `mariadb:11` as you will not know the compatible versions in the future.

Reply:
Then I would also have to update it in each version of the documentation in the future. I think chances are quite good we will keep suporting the newest versions of mariadb in the Core, while outdated versions tend to not work anymore one day. If this changes we can still use a different image

Reply:
IMO mariadb:11 is not officially supported at all.

Reply:
changed it to mariadb:latest